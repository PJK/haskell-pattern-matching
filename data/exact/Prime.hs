f :: Int -> Int
f x | 5 > 1 && isPrime 2 = 1
