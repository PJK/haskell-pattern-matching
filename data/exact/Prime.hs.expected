{
    "tag": "AnalysisSuccess",
    "contents": [
        [
            "f",
            {
                "tag": "NonExhaustive",
                "contents": [
                    [
                        [
                            {
                                "tag": "WildcardPattern",
                                "contents": []
                            }
                        ],
                        "Constraints:\n  fresh2 == False\n  fresh2 == 5 > 1 && isPrime 2\n\nSatisfiable. Model:\n  fresh2    = False :: Bool\n  isPrime 2 = False :: Bool\n"
                    ]
                ]
            }
        ]
    ]
}
